src-tauri/Cargo.toml

```diff
@@ -44,8 +44,10 @@ http = "1"
 log = "0.4.21"
 rand = "0.8.5"
 regex = "1.10.2"
-reqwest = { workspace = true, features = ["multipart", "cookies", "gzip", "brotli", "deflate", "json", "native-tls-alpn"] }
+reqwest = { workspace = true, features = ["multipart", "cookies", "gzip", "brotli", "deflate", "json", "rustls-tls"] }
```

Comment:
I think you could save some dependencies by picking `rustls-tls-manual-roots-no-provider` instead of the default `rustls-tls` (which pulls in webpki-roots).

Reply:
Oh, good one! I didn't notice that one in my autocomplete initially 🙏🏻